app/javascript/styles/mastodon/variables.scss

```diff
@@ -26,7 +31,7 @@ $ui-base-color: $classic-base-color !default;                  // Darkest
 $ui-base-lighter-color: lighten($ui-base-color, 26%) !default; // Lighter darkest
 $ui-primary-color: $classic-primary-color !default;            // Lighter
 $ui-secondary-color: $classic-secondary-color !default;        // Lightest
-$ui-highlight-color: $classic-highlight-color !default;        // Vibrant
+$ui-highlight-color: #2b5fd9;
```

Comment:
Why this?

Reply:
This is to acquire contrast ratio between white texts and background as `$ui-highlight-color`,. Without this, the contrast ratio is 3.45 (at least 4.5 is required).
I set `$highlight-text-color` as `$classic-highlight-color` and apply this to all texts, loading bar, and so on, so changes are only background color of buttons with text.